README.md

```diff
@@ -485,10 +485,10 @@ You can find examples in the [examples/](examples/) directory.
 We are happy to help, please contact us or create an issue.
 
 - [Twitter: @getAlby](https://twitter.com/getAlby)
-- [Telegram group](https://t.me/getAlby)
-- support at getalby.com
+- [Telegram Community Chat](https://t.me/getAlby)
+- e-mail to support at getalby.com
```

Comment:
can we add support email here as well

Reply:
![image](https://github.com/getAlby/js-sdk/assets/16482809/b6848d7a-71f7-419c-8e98-2fa0e2d3b841)
i guess this is it, maybe for bots we didn't want to put it straight support@getalby.com

Reply:
@itstomekk I think it's fine, it's public on our website and you just leaked it to github as a comment anyway :D